Cargo.toml

```diff
@@ -63,3 +63,15 @@ lto = true
 
 [profile.release.build-override]
 opt-level = 3
+
+[patch."https://github.com/gakonst/ethers-rs"]
```

Comment:
The build is breaking because foundry-cli uses the latest ethers (v2.0.9) which has a dependency conflict with cargo-dependencies uses in cargo-risczero

<img width="1334" alt="Screen Shot 2023-08-24 at 7 49 16 PM" src="https://github.com/risc0/risc0/assets/6301222/b38ce14e-7fa9-40a6-b586-da207751875a">

Reply:
I think with #810 now merged this should no longer be an issue. Sorry for introducing such a radioactive dependency to the repo 😅